dockers/docker-fpm-frr/frr/bgpd/templates/internal/instance.conf.j2

```diff
@@ -0,0 +1,34 @@
+!
+! template: bgpd/templates/internal/instance.conf.j2
+!
+  neighbor {{ neighbor_addr }} remote-as {{ bgp_session['asn'] }}
+  neighbor {{ neighbor_addr }} description {{ bgp_session['name'] }}
+!
+{% if neighbor_addr | ipv4 %}
+  address-family ipv4
+    neighbor {{ neighbor_addr }} peer-group INTERNAL_PEER_V4
+!
+{%   if CONFIG_DB__DEVICE_METADATA['localhost']['sub_role'] == 'BackEnd' %}
+    neighbor {{ neighbor_addr }} route-map FROM_BGP_INTERNAL_PEER_V4 in
```

Comment:
Is route-map configuration line repeated?  The same line present in the peer_group.conf.j2 configuration

Reply:
I looked at both .. they are not same.
Here we set it per neighbor_ip_address, while in the peer_group.conf.j2 it is set on the peer group -- similar to how it was done before as well.